WordPress/WordPressTest/BloggingPromptsServiceTests.swift

```diff
@@ -403,31 +373,27 @@ private extension BloggingPromptsServiceTests {
         return try XCTUnwrap(passedParameters[key], "Param not found for \(key)")
     }
 
-    func passedNumber() throws -> Int? {
-        return try passedParameter("per_page") as? Int
+    func passedNumber() throws -> Int {
+        return try XCTUnwrap(passedParameter("per_page") as? Int)
     }
 
-    func passedDate() throws -> Date {
+    func passedDate() throws -> String {
```

Comment:
We were previously converting the passed parameter from a string into a date, but we start with a stringified date to do the comparison. By using stringified dates for both sides, it's way easier to do comparison